src/Tizen.Messaging/Tizen.Messaging.Email/EmailSender.cs

```diff
@@ -40,35 +44,39 @@ public static async Task<EmailSendResult> SendAsync(EmailMessage email)
             email.FillHandle();
             email.Save();
 
-            Interop.Email.EmailSentCallback _emailSendingCallback = (IntPtr handle, int result, IntPtr userData) =>
+            int id = _callbackId++;
+            _sendCbMap[id] = (IntPtr handle, int result, IntPtr userData) =>
             {
-                task.SetResult((EmailSendResult)result);
+                task?.SetResult((EmailSendResult)result);
+                _sendCbMap.Remove((int)userData);
             };
 
-            ret = Interop.Email.SetCb(email._emailHandle, _emailSendingCallback, IntPtr.Zero);
+            ret = Interop.Email.SetCb(email._emailHandle, _sendCbMap[id], (IntPtr)id);
```

Comment:
SetCb is set a global callback for recevice result of sent.
but, it could be overwittten by next method call that was called before complete a previous request.

Reply:
All the callbacks are differentiated with id , so they will not interfere each other.

Reply:
Ok, it stored with mail id and it can save a multiple mail id